vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -127,13 +132,16 @@ func main() {
 	})
 	clientset := getClient()
 	server := &http.Server{
-		Addr:      fmt.Sprintf(":%d", *port),
-		TLSConfig: configTLS(clientset, certs.serverCert, certs.serverKey),
+		Addr: fmt.Sprintf(":%d", *port),
 	}
+	// this will check if there are new certs before every tls handshake
```

Comment:
Actually, this will return up-to-date cert before every handshake, right?

Reply:
yeah, that's correct, I've updated the comment :)